packages/app/components/drop/drop-music.tsx

```diff
@@ -222,6 +222,17 @@ export const DropMusic = () => {
     previousIsSaveDrop.current = isSaveDrop;
   }, [isSaveDrop, trigger]);
 
+  useEffect(() => {
+    resetDropState();
+  }, [resetDropState]);
```

Comment:
Is this intended?

Reply:
yes, I have moved the drop creation state to context, so we need to reset it when the component mounts. Moved it to context to prevent some weird issues in redirection from checkout to drop form. Also since one drop is allowed at any moment, I think having a single context can be useful if we decide to show some indicator elsewhere similar to claim for e.g.